README.md

```diff
@@ -3,21 +3,39 @@
 1) build and test scripts are run
 2) git config are setup with username Bhoos Action and email action@bhoos.com
 3) package version is updated according to branch name i.e minor or patch so this workflow should be only triggered from those branches
-4) setup to publish the package using github token provided from env GITHUB_PERSONAL_ACCESS_TOKEN, then  package is published
+4) setup to publish the package using github token provided from env GITHUB_PERSONAL_ACCESS_TOKEN,  also add NODE_AUTH_TOKEN else there will be authentication issue then package is published with updated version and next tag
 5) the update is then pushed to the current branch and the master branch
-6) tags named nextv(nextVersion) is pushed
-7) a draft release is created with above tag
+6) a draft release is created with above version
+7) and upon release the said version is moved to latest tag
 
 
-## Example USAGE
-    on:
-      push:
-        branches:
-          - minor
-          - patch
-        .....
-        .....
-    - name: Use bhoos action
-        uses: actions/bhoos-package-action@{currentVersionofTag}
-        env:
-          GITHUB_PERSONAL_ACCESS_TOKEN: ${{ github.token }}
+## Example usage
+
+```bash
+  on:
+    push:
+      branches:
+        - minor
+        - patch
+  release:
+    types: [published]
+  jobs:
+    package-release:
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v2
+          with:
+            fetch-depth: 0
+        - name: Use Node.js ${{ matrix.node-version }}
+          uses: actions/setup-node@v1
+          with:
+            node-version: '10.x'
+        - name: Use Bhoos release action
+          uses: bhoos/action-release@v{currentVersion}
+          env:
+            # make sure github.token instead of personal access token is passed else there will be an infinite loop
+            GITHUB_PERSONAL_ACCESS_TOKEN: ${{ github.token }}
```

Comment:
@karkipy there is no point in providing github.token to our action which can get this from github context from within